### PR TITLE
Cleaned up Upstart scripts and added systemd scripts

### PR DESCRIPTION
**Pull request by pidydx**
[Review on Reviewable](https://reviewable.io/reviews/google/grr/215)

### CODE REVIEW

File: config/centos/default/grr-http-server [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# Configuration file for grr-http-server start-up script
```
Comment (destijl) [inline config/centos/default/grr-http-server]: So what's happening with these centos files? I think you removed them to use the ones from the os-agnostic upstart directory. But I don't see any corresponding build code change for rpms (only debs). How does the rpmbuilder know where to find them now?

Comment (pidydx) [inline config/centos/default/grr-http-server]: I actually put those files there in the first place back in Oct just as storage so people could find server startup scripts that would work with Centos/RHEL if they ran the servers on non-Ubuntu Linux.  The rpmbuilder is only dealing with the client stuff so it only cares about  grr-client.initd.in

File: config/upstart/grr-dataserver-master.conf [hunk 2]
```diff
@@ -10,11 +10,15 @@ kill timeout 300
 
 pre-start script
 	mkdir -p /var/log/grr
+	mkdir -p /var/run/grr/tmp/dataserver
```
Comment (destijl) [inline config/upstart/grr-dataserver-master.conf]: Lets add -master and -slave to all of these paths. It's possible (but not super useful) to run them on the same machine.

File: config/centos/default/grr-http-server (hunk 1 above)
Comment (destijl) [inline config/centos/default/grr-http-server]: Ah OK thanks.

**Comment by destijl** [comment]
LGTM

File: config/upstart/grr-dataserver-master.conf (hunk 2 above)
Comment (pidydx) [inline config/upstart/grr-dataserver-master.conf]: Done!

**Pull request merged**